### DISCUSSION TITLE
Convert prisma.schema to nexus.schema or vice versa

### DISCUSSION THREAD

**Discussion by ralvs** [prisma/prisma#2897] [Q&A] [answered]
Is there a way to convert one schema file to another?
Because the only way I could find is, I have to write or introspect the prisma.schema and then I have to write all the nexus.schema

Thanks

**Comment by ryands17** [comment, marked as answer]
Hey @ralvs :wave: 
I don't think there is any way to convert from Nexus to Prisma, but from Prisma to Nexus you can use a community made package [here](https://paljs.com/) that would help you to autogenerate CRUD functionality from the Prisma model.

**Reply by ralvs** [reply]
Wow, that looks nice!
Will test it.
Thanks @ryands17

**Comment by ludralph** [comment]
Hi @ralvs,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
